codegenerator/cli/src/config_parsing/human_config.rs

```diff
@@ -440,6 +440,12 @@ pub mod evm {
         )]
         #[serde(skip_serializing_if = "Option::is_none")]
         pub name: Option<String>,
+        #[serde(skip_serializing_if = "Option::is_none")]
+        #[schemars(
+            description = "An object representing additional fields to add to the event passed to \
```

Comment:
Didn't you say in the video that it overrides rather than adds to?

Reply:
Yeah, makes sense to adjust the copy. I just copy pasted it from the global one.